pom.xml

```diff
@@ -267,6 +267,37 @@
           </systemProperties>
         </configuration>
       </plugin>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-enforcer-plugin</artifactId>
+        <version>1.4.2.jenkins-1</version>
+        <configuration>
+          <rules combine.children="append">
+            <requireUpperBoundDeps>
+              <excludes>
+                <!-- TODO Stapler requests 14 while Jenkins requests 11; should backdate the Stapler version -->
+                <exclude>com.google.guava:guava</exclude>
+                <!-- TODO jenkins-test-harness-htmlunit requests 1.2, newer than 1.1.x in core -->
+                <exclude>commons-logging:commons-logging</exclude>
+                <!-- Stapler requests 2.x while core requests 1.x; anyway not used at runtime -->
+                <exclude>com.google.code.findbugs:jsr305</exclude>
+                 <!-- needed between Jenkins 2.36—2.60 -->
+                <exclude>org.kohsuke:access-modifier-annotation</exclude>
+                 <!-- core requests 1.4, sshd requests 1.2 -->
+                <exclude>org.jenkins-ci.modules:instance-identity</exclude>
```

Comment:
Rather than excluding this violation, fix it, perhaps by upgrading `sshd-module` appropriately.